<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>723</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>624</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background: #333333;
    width: 10px; /* when vertical */
    height: 10px; /* when horizontal */
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>1762</width>
        <height>882</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QGraphicsView{
	background-color: white;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="styleSheet">
         <string notr="true">QLCDNumber{
	color: white;
    background-color: black;
	border: 1px solid red;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>New Game</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pauseButton">
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="highscoreButton">
        <property name="text">
         <string>Highscores</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber_1">
        <property name="styleSheet">
         <string notr="true">QLCDNumber{
    color: white;    
    background-color: black;
	border: 1px solid red;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QMenuBar {
     background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 lightgray, stop:1 darkgray);
 }

 QMenuBar::item {
     spacing: 3px; /* spacing between menu bar items */
     padding: 1px 4px;
     background: transparent;
     border-radius: 4px;
 }

 QMenuBar::item:selected { /* when selected using mouse or keyboard */
     background: #a8a8a8;
 }

 QMenuBar::item:pressed {
     background: #888888;
 }</string>
   </property>
   <widget class="QMenu" name="menuSound">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionMute"/>
    <addaction name="actionImmortal"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGame_Rules"/>
    <addaction name="actionGame_Controls"/>
   </widget>
   <addaction name="menuSound"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionMute">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/images/29_Sound_alt-128_10x10.png</normaloff>
     <normalon>:/images/30_Sound_off-128_10x10.png</normalon>:/images/29_Sound_alt-128_10x10.png</iconset>
   </property>
   <property name="text">
    <string>Mute</string>
   </property>
  </action>
  <action name="actionImmortal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/images/sign-off-128_10x10.png</normaloff>
     <normalon>:/images/sign-check-128_10x10.png</normalon>:/images/sign-off-128_10x10.png</iconset>
   </property>
   <property name="text">
    <string>Immortal Mode</string>
   </property>
   <property name="toolTip">
    <string>Immortal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionGame_Rules">
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/images/help-128_10x10.png</normaloff>:/images/help-128_10x10.png</iconset>
   </property>
   <property name="text">
    <string>Game Rules</string>
   </property>
  </action>
  <action name="actionGame_Controls">
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/images/controller.png</normaloff>:/images/controller.png</iconset>
   </property>
   <property name="text">
    <string>Controls</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="recources.qrc"/>
 </resources>
 <connections/>
</ui>
